agents/rust/mediator/mediation/.github/workflows/rust-basic.yml

```diff
@@ -0,0 +1,130 @@
+name: Rust basic workflow
+
+on:
+  push:
+    branches: [ "main", "ci-integration-test", "ci-cd", "dev", aries-vcx-mediator-integration]
+  pull_request:
+    branches: [ "main", "ci-integration-test", "ci-cd", "dev", aries-vcx-mediator-integration]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  check:
+    name: Check for obvious errors
```

Comment:
There's workspace wide job `workspace_clippy`  which runs clippy on all projects in workspace. Running `cargo check` does subset of what clippy does. So this is somewhat superfluous and I think can be dropped in favor of `workspace_clippy`